RecipesBase/src/RecipesBase.jl

```diff
@@ -87,7 +87,7 @@ function get_function_def(func_signature::Expr, args::Vector)
     elseif func_signature.head ≡ :call
         func = Expr(
             :call,
-            :(RecipesBase.apply_recipe),
+            :($RecipesBase.apply_recipe),
```

Comment:
```suggestion
            :($apply_recipe),
```

We tries that, but it failed, right ?

Reply:
Yes. This is a definition, so we need indirect interpolation because of JuliaLang/julia#47726.
In all other cases the more direct approach works (note that all tests pass on [de5a423](https://github.com/JuliaPlots/Plots.jl/pull/4559/commits/de5a423e7a5911fdca2fdae117c4f2db80539b29))

Reply:
In this case, and because this is a low-level julia detail (maybe bug, I'm not competent to say it is), I would prefer to be explicit and consistent, instead of being implicit and kindof obscure.
IMO, it is not immediately obvious to the reader of `RecipesBase` code why you can use `:($is_key_supported)` in some place and not `:($apply_recipe)`.

Reply:
Side note: [Explicit is better than implicit](https://peps.python.org/pep-0020/#the-zen-of-python), taking the good bits from `Python` ...